nursery/capture-webcam-video.yml

```diff
@@ -0,0 +1,25 @@
+rule:
+  meta:
+    name: capture webcam video
+    namespace: collection/webcam
+    author: "@johnk3r"
+    description: Rule that detects a system's webcam being used to capture video
+    scope: function
+    att&ck:
+      - Collection::Video Capture [T1125]
+  features:
+    - or:
+      - and:
+        - api: capCreateCaptureWindow
+        - basic block:
+          - and:
+            - api: SendMessage
+            - number: 0x43E = WM_CAP_SEQUENCE
+        - basic block:
+          - and:
+            - api: SendMessage
+            - number: 0x417 = WM_CAP_FILE_SAVEAS
+      - basic block:
+        - and:
+          - api: SendMessage
+          - number: 0x414 = WM_CAP_FILE_SET_CAPTURE_FILE
```

Comment:
```suggestion
    - and:
      - os: windows
      - api: capCreateCaptureWindow
      - basic block:
        - and:
          - api: SendMessage
          - number: 0x43E = WM_CAP_SEQUENCE
      - or:          
        - basic block:
          - and:
            - api: SendMessage
            - number: 0x417 = WM_CAP_FILE_SAVEAS
        - basic block:
          - and:
            - api: SendMessage
            - number: 0x414 = WM_CAP_FILE_SET_CAPTURE_FILE
```

> ... it seems like we could look for calling capCreateCaptureWindow + calling capCaptureSequence + (calling capFileSaveAs or calling capFileSetCaptureFile)

Reply:
two changes:
  - restrict to `os: windows`
  - use logic from original issue, unless you have a better idea?

Reply:
1st I had forgotten about the "SO" filter. Thanks for reinforcing.
2nd Your logic got better. Thanks for the sugestion.